packages/hebao_v2/contracts/lib/ReentrancyGuard.sol

```diff
@@ -11,14 +11,14 @@ pragma solidity ^0.7.0;
 ///      the number of times the value is changed
 contract ReentrancyGuard
 {
-    //The default value must be 0 in order to work behind a proxy.
-    uint private _guardValue;
+    //The default value must be 1 in order to work behind a proxy.
+    uint private _guardValue = 1;
```

Comment:
I don't think this will work because the storage variable in the proxy contract will not be initialized to 1, only the storage variable in the implementation contract will have it set but that won't be used.

Reply:
Sorry, I didn't pay enough attention to the comment line above. all changes in ReentrancyGuard had been reverted.